docs/ref/settings.txt

```diff
@@ -963,6 +963,23 @@ TEST_TBLSPACE_TMP
     Use the :setting:`TBLSPACE_TMP <TEST_TBLSPACE_TMP>` entry in the
     :setting:`TEST <DATABASE-TEST>` dictionary.
 
+.. setting:: DATA_UPLOAD_MAX_MEMORY_SIZE
+
+DATA_UPLOAD_MAX_MEMORY_SIZE
+---------------------------
+
+.. versionadded:: 1.8
+
+Default: ``2621440`` (i.e. 2.5 MB).
```

Comment:
I'd like to consider changing the unit to `MiB`. But that's out of the scope of the PR. I'll bring it up on the django-developers mailing list.